### DISCUSSION TITLE
Error CS0246

### DISCUSSION THREAD

**Discussion by oblist7** [serenity-is/Serenity#6489] [Help] [answered]
Hello, what can be done?)
Master/Detail Editing  |  MovieTutorial  |  MovieCast 

Error CS0246
The type or namespace name 'RecipeCompositionEditorAttribute' could not be found (are you missing a using directive or an assembly reference?)	..\Modules\RecipeDB\Recipe\RecipeForm.cs

[RecipeCompositionEditor, IgnoreName]

**Comment by oblist7** [comment]
pls help me )
![image](https://user-images.githubusercontent.com/60620893/183293904-190c99d1-3b00-4599-a024-5aa433c69fdb.png)

**Reply by haajit** [reply]
Hi
Whate is RecipeCompositionEditor namespace?
Please send source of RecipeCompositionEditor

**Comment by oblist7** [comment, marked as answer]
Hi, 
thx Mr. Hassan Kashani.
Seems to have solved the issue, 
![image](https://user-images.githubusercontent.com/60620893/183401922-657689ed-dc7c-491a-88ed-4d65574b37a5.png)
manually added a class here

**Reply by stixoffire** [reply]
When you get this error it is because you need to build the typescript T4 templates.
That is what creates those for you - automagically..
